api-server/web-server/src/api/v2.rs

```diff
@@ -126,6 +126,7 @@ pub fn routes<
     let router = router
         .route("/token", get(token_ids))
         .route("/token/:id", get(token))
+        .route("/token/:id/transactions", get(token_transactions))
```

Comment:
Plz rebase this PR on top of `https://github.com/mintlayer/mintlayer-core/pull/1993` and add/update a changelog entry, mentioning that a new endpoint was added.

P.S. IMO just mentioning the new endpoint is enough for the changelog, e.g. [here is how I mentioned new endpoints for 1.0.2](https://github.com/mintlayer/mintlayer-core/blob/add_changelogs/api-server/CHANGELOG.md#added-2). Yours should go into the "Unreleased" section at the top.